cmd/ormb-storage-initializer/cmd/pull-and-export.go

```diff
@@ -110,42 +107,11 @@ func relayoutModel(modelDir string) error {
 	if err != nil {
 		return err
 	}
-	destinationDir, err := filepath.Abs(modelDir)
-	if err != nil {
-		return err
-	}
 
-	// For TensorFlow serving, MUST move /mnt/models/model to /mnt/models/1 (dstDir).
-	ormbfileBytes, err := ioutil.ReadFile(path.Join(modelDir, "ormbfile.yaml"))
-	if err != nil {
+	if err := os.Rename(originalDir, path.Join(modelDir, "1")); err != nil {
```

Comment:
Can we keep 1 in the model content layer?

Reply:
I think it is OK. the directory structure will be as follow when ormb push:
```
--| models
    --| model
       --| 1
           --| xxx.pb
    --| ormbfile.yaml
```

Reply:
Yeah, in this way we do not need to have this trick. As you know, other formats such as onnx do not need 1 dir.